### PR TITLE
chore: remove pr labeler requirement

### PR DESCRIPTION
**Pull request by alextran1502**
Remove pr label enforcing job since it is now done through CI

### CODE REVIEW

**Review by bo0tzz — DISMISSED** [review]
I think we should still keep this job as an enforcement in case the other one fails, as well as if the changes don't result in that one automatically applying a label.

**Comment by alextran1502** [comment]
You will get no green checkmark automatically though

**Comment by zackpollard** [comment]
> I think we should still keep this job as an enforcement in case the other one fails, as well as if the changes don't result in that one automatically applying a label.

The two don't work together, the label gets applied too late and then the label check job doesn't run because it won't be triggered by the bot adding a label.

**Comment by bo0tzz** [comment]
Ah, because the workflow_run won't be triggered due to that annoying loop prevention, ugh... Any ideas for how we can work around that? If we can I think it would still be worth having this enforcement. As an alternative, maybe the enforcement can be baked into the other job?

**Comment by zackpollard** [comment]
> Ah, because the workflow_run won't be triggered due to that annoying loop prevention, ugh... Any ideas for how we can work around that? If we can I think it would still be worth having this enforcement. As an alternative, maybe the enforcement can be baked into the other job?

We talked about it earlier and really it's not a big deal if it doesn't get applied, we can always sort any missing a label when we come to making the release notes

**Review by zackpollard — APPROVED** [review]

**Comment by bo0tzz** [comment]
> we can always sort any missing a label when we come to making the release notes

Definitely true, but IMO it's much easier to do that when the context of the PR is still fresh.

File: .github/workflows/pr-require-label.yml [hunk 1]
```diff
@@ -1,17 +0,0 @@
-name: Enforce PR labels
-
-on:
-  pull_request:
-    types: [labeled, unlabeled, opened, edited, synchronize]
-  workflow_run:
-    workflows: ["Pull Request Labeler"]
-    types:
-      - completed
```
Comment (roschaefer) [inline .github/workflows/pr-require-label.yml:9]: This very condition was supposed to re-run the "Enforce PR labels" after "Pull Request Labeler". At least, that's how it worked in my fork.

So what's different? In the PR discussion a [loop prevention](https://github.com/immich-app/immich/pull/10081#issuecomment-2158738753) is mentioned. Does that prevent the desired behaviour?

Comment (zackpollard) [reply .github/workflows/pr-require-label.yml:9]: Not sure really, I think we basically decided it wasn't worth investigating when the labels aren't actually "mandatory" they're just a really nice help and the other action should catch most cases anyway

Comment (roschaefer) [reply .github/workflows/pr-require-label.yml:9]: Fair enough. When I implemented it this way, I felt a little dirty about the dependency across workflows. Less complexity is a good thing, too.

**Pull request merged**